pages_builder/pages/notification-banner.yml

```diff
@@ -27,3 +27,12 @@ examples:
     title: Destructive banner
     type: destructive
     message: <a href="#">With a link</a> and some plain text but without an action
+  -
+    title: Temporary message banner
+    type: temporary-message
+    heading: >
+            <h2> 
```

Comment:
We should avoid passing HTML to patterns wherever possible (and I can’t see what the `history-status-date` class is being used for.